src/SFA.DAS.Charities.Database/Tables/Charity.sql

```diff
@@ -0,0 +1,24 @@
+﻿CREATE TABLE [dbo].[Charity]
+(
+    [Id] INT NOT NULL PRIMARY KEY, 
+    [Name] VARCHAR(MAX) NOT NULL, 
+    [CompanyNumber] VARCHAR(MAX) NOT NULL, 
+    [RegisteredCharityNumber] INT NOT NULL, 
```

Comment:
It might be overkill, but would you agree that an unique constraint would be beneficial here?

Reply:
Don't think there is any benefit of doing that as there is no manual entry and only imported data.